about.css

```diff
@@ -0,0 +1,23 @@
+
+
+
+
+@media only screen and (max-width: 635px) {
+    .nav-wrap{
+        width: 66.6em;
+    }
+    footer{
+        position: absolute;
+        width: 63.6em;
+        bottom: -75em;
+    }
+    .what-wrap{
+        padding-left: 1rem;
+    }
+    .how-wrap{
+        padding-left: 1rem;
+    }
+    .how-wrap iframe{
```

Comment:
Hi Kunal, Please check if this code is redundant as padding is already applied on .how-wrap.